lib/chai/core/assertions.js

```diff
@@ -1172,6 +1172,11 @@ module.exports = function (chai, _) {
     } else {
       actual = Object.keys(obj);
 
+      if (typeof Object.getOwnPropertySymbols === 'function') {
+        // Object.keys excludes Symbols so we need this too
+        actual = actual.concat(Object.getOwnPropertySymbols(obj));
```

Comment:
Slight concern with this - `Object.getOwnPropertySymbols` returns enumerable _and_ non-enumerable keys - somewhat changing the semantics of what this assertion already does. Should we filter out the non-enumerable keys? Something like:


```js
actual = actual.concat(Object.getOwnPropertySymbols(obj).filter(function (symbol) {
  return Object.getOwnPropertyDescriptor(obj, symbol).enumerable;
}));
````